sycl/unittests/kernel-and-program/device/CMakeLists.txt

```diff
@@ -0,0 +1,3 @@
+add_sycl_unittest_with_device(KernelAndProgramTestsWithDevice OBJECT
```

Comment:
Thinking out loud: would it be interesting to have an option to test the runtime part which doesn't have dependency on clang? I mean to have a switch disabling these `sycl_unittest_with_device` tests to seed-up local development.

Reply:
I don't think it's needed. We currently have a single unit test that has dependency on clang and only because it needs the presence of *any* device image, even an empty one (otherwise we get a failure in ProgramManager since no image can be found). I think this dependency shouldn't be hard to get rid of and we should do away with such unit tests entirely.